awx_collection/plugins/modules/tower_user.py

```diff
@@ -114,48 +122,64 @@
 def main():
     # Any additional arguments that are not fields of the item can be added here
     argument_spec = dict(
-        username=dict(required=True),
-        first_name=dict(),
-        last_name=dict(),
-        password=dict(no_log=True),
-        email=dict(required=False, default=''),
-        superuser=dict(type='bool', default=False),
-        auditor=dict(type='bool', default=False),
+        username=dict(required=True, type='str'),
+        first_name=dict(required=False, type='str'),
+        last_name=dict(required=False, type='str'),
+        email=dict(required=False, type='str'),
+        is_superuser=dict(required=False, type='bool', default='False', aliases=['superuser']),
+        is_system_auditor=dict(required=False, type='bool', default='False', aliases=['auditor']),
+        password=dict(required=False, type='str', default=''),
         state=dict(choices=['present', 'absent'], default='present'),
     )
 
+# TODO: Implement a way to also generate proper aliases for backwards compatibility
+# and also add the aliases to documentation!
```

Comment:
aliases are being used, I think this can be removed??

Reply:
Woops, good catch!  Sorry about that!